ngraph/frontend/tensorflow/src/op/select.cpp

```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```

Comment:
line 19 add node name to Error message to simplify debug and why operation cannot be converted